tests/end2end/features/sessions.feature

```diff
@@ -367,3 +367,24 @@ Feature: Saving and loading sessions
   Scenario: Loading a session which doesn't exist
     When I run :session-load inexistent_session
     Then the error "Session inexistent_session not found!" should be shown
+
+
+    # Test load/save of pinned tabs
```

Comment:
on this note, is there a reason this file is indented with a size of 2? The others seem to use a size of 4, and I found that kind of weird. (I don't think I should reindent it in this PR though, that would get too messy).